content/docs/command-reference/commit.md

```diff
@@ -34,8 +34,7 @@ these step(s) during the process of tracking each file or directory:
 - Store the file contents in the cache.
 
 > Skipping these steps is typically done to avoid caching unfinished data, for
-> example when exploring different data or
-> [stages](/doc/command-reference/run)).
+> example when [experimenting](/doc/command-reference/exp).
```

Comment:
```suggestion
> example when [experimenting](/doc/user-guide/experiment-management).
```

Reply:
But I'm not sure we want to plug experiments here. I don't remember `commit` used much if at all in any exp ref or guide. Probably roll back

Reply:
They'll need when  using `--no-commit` flag to `dvc exp run`. For the singular use, `exp run` is no different from `dvc repro`, it leaves artifacts in the workspace and those may be committed or not.

Reply:
It's different. It creates an experiment (Git custom ref. etc.) If this is not about experimentation let's use non-exp concepts.

Reply:
Agreed, it's also about `dvc add` which might be used outside experiments.

Reply:
I replace `repro` with `exp` here, because the sentence mentions "unfinished data." Usually we have "unfinished data" while experimenting. I can add "or [running pipelines] (repro)" here, or remove this sentence completely. 

`dvc add` is mentioned in the paragraph one above.

Reply:
> Usually we have "unfinished data" while experimenting.

Sure, "experimenting" in a very general sense, as in "trying things out". That's still not the same as a data science experiment (param tuning, etc.)

> I can add "or [running pipelines] (repro)" here, or remove this sentence completely.I don't think it's a problem to mention "unfinished data" here without associating it with `dvc exp`.

>

Reply:
OOPS FIXING PREVIOUS COMMENT:

> I can add "or [running pipelines] (repro)" here, or remove this sentence completely.

I don't think it's a problem to mention "unfinished data" here without associating it with `dvc exp`. And not sure I see how running pipelines relates exactly to `--no-commit` -- you can still run it right? But again that's a jump away from the topic I think. We can just roll this link back IMO.

Reply:
Removed it. PTAL. @jorgeorpinel

Reply:
Thanks.